src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -890,6 +890,7 @@ public sealed partial class JsonSerializableAttribute : System.Text.Json.Seriali
         public string? TypeInfoPropertyName { get { throw null; } set { } }
         public System.Text.Json.Serialization.JsonSourceGenerationMode GenerationMode { get { throw null; } set { } }
     }
+    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Is this expected?

cc @layomia @dotnet/area-system-text-json

Reply:
ping @layomia @dotnet/area-system-text-json

Reply:
I believe it is https://github.com/dotnet/runtime/blob/e652e007df4b7e6a66dbdc4e981496537bd0fcc1/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerContext.cs#L12

Presumably since the class is only meant to be inherited by the source generator.

Reply:
> Presumably since the class is only meant to be inherited by the source generator.

But we have other APIs that take a `JsonSerializerContext` object. And we have `public` methods on it:

https://github.com/dotnet/runtime/blob/385fc8832b1d4160abe13645054b65458b6a12f8/src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Get.cs#L46

https://github.com/dotnet/runtime/blob/385fc8832b1d4160abe13645054b65458b6a12f8/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerContext.cs#L26

IMO, it is odd that a type we expect users to interact with is marked with `EditorBrowsableState.Never`.

Reply:
I don't think we want that attribute at all. cc @layomia

Reply:
I removed the attribute from both the ref and the source.